.github/workflows/build.yml.in

```diff
@@ -202,6 +199,9 @@ jobs:
           MATHLIB_CACHE_SAS: ${{ secrets.MATHLIB_CACHE_SAS }}
           MATHLIB_CACHE_S3_TOKEN: ${{ secrets.MATHLIB_CACHE_S3_TOKEN }}
 
+      - name: update {Mathlib, Tactic, Counterexamples, Archive}.lean
```

Comment:
I do wonder, however, if this step will do any work.
If it operates on the local branch (modified compared to what github had), it will succeed. Can you either test and comment if it does work, or remove this step, please?